sql/mob_skills.sql

```diff
@@ -438,7 +438,7 @@ INSERT INTO `mob_skills` VALUES (630,374,'Kick_Out',4,10.0,2000,1500,4,0,0,0);
 INSERT INTO `mob_skills` VALUES (631,375,'Shock_Wave',4,7.0,2000,1500,4,0,0,2);
 INSERT INTO `mob_skills` VALUES (632,376,'Flame_Armor',0,7.0,2000,1500,1,0,0,0);
 INSERT INTO `mob_skills` VALUES (633,354,'Howl',1,20.0,2000,1500,1,0,0,0);
--- INSERT INTO `mob_skills` VALUES (634,378,'Meteor',0,7.0,2000,1500,4,0,0,0);
+-- INSERT INTO `mob_skills` VALUES (634,2256,'Meteor',0,7.0,2000,1500,4,0,0,0);
```

Comment:
not sure who uses a mobskill version of meteor that just looks like a
regular meteor..

On Wed, May 11, 2016 at 11:08 PM, TeoTwawki <notifications@github.com>
wrote:

> In sql/mob_skills.sql
> <https://github.com/DarkstarProject/darkstar/pull/3167#discussion_r62965670>
> :
>
> > @@ -438,7 +438,7 @@ INSERT INTO `mob_skills` VALUES (630,374,'Kick_Out',4,10.0,2000,1500,4,0,0,0);
> >  INSERT INTO `mob_skills` VALUES (631,375,'Shock_Wave',4,7.0,2000,1500,4,0,0,2);
> >  INSERT INTO `mob_skills` VALUES (632,376,'Flame_Armor',0,7.0,2000,1500,1,0,0,0);
> >  INSERT INTO `mob_skills` VALUES (633,354,'Howl',1,20.0,2000,1500,1,0,0,0);
> > --- INSERT INTO `mob_skills` VALUES (634,378,'Meteor',0,7.0,2000,1500,4,0,0,0);
> > +-- INSERT INTO `mob_skills` VALUES (634,2256,'Meteor',0,7.0,2000,1500,4,0,0,0);
>
> Amended now. The other animation ID I had is for animating a regular non *hump
> the ground and die*, use of meteor.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/DarkstarProject/darkstar/pull/3167/files/a986c64f0360e1c4effea4c48ff7f47da327a901#r62965670>
>